src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -164,8 +165,12 @@ int replicationFactor() {
     public void run() {
       try {
         if (_topicCreationEnabled) {
-          _topicFactory.createTopicIfNotExist(_zkConnect, _topic, _configuredTopicReplicationFactor,
-              _partitionsToBrokerRatio, new Properties());
+          try {
+            _topicFactory.createTopicIfNotExist(_zkConnect, _topic, _configuredTopicReplicationFactor,
+                _partitionsToBrokerRatio, new Properties());
+          } catch (TopicExistsException e) {
```

Comment:
Good point. I saw the exception trace in the log but I missed the fact that it is already caught.

Then I think we can reduce the log level from INFO to DEBUG if the topic has already been created. The idea is that user should not see exception trace in the log if user starts KafkaMonitor without first creating the topic.